backend/src/test/java/com/redhat/cloud/notifications/routers/EndpointResourceTest.java

```diff
@@ -2055,18 +2056,20 @@ void testEndpointTest() {
         given()
             .header(identityHeader)
             .when()
+            .contentType(JSON)
```

Comment:
I was forced to specify the content type as `application/json` even if those requests contained no body, as otherwise I would get a "415 Unsupported Media Type" error back from Quarkus. Is there a way to avoid this?

Reply:
We depend on `quarkus-resteasy-reactive-jackson` which makes `application/json` the expected content type for all inputs and outputs. I don't think there's an issue here though.